### DISCUSSION TITLE
[Feature] 部分想法与建议

### DISCUSSION THREAD

**Discussion by YiHui-Liu** [walinejs/waline#946] [Ideas]
1. 在后台增加根据页面筛选评论的功能
2. 为什么评论显示的时间需要使用`insertedAt`，置顶也会改变这个，导致前端评论显示的时间会改变，出现回复竟然比评论时间还要早的情况

**Comment by lizheming** [comment]
1. 可以提个 issue 后续支持一下
2. 修改置顶的时候不会修改 insertedAt 字段才对。评论显示根据 insertedAt 主要是同步了 Valine 的逻辑。

**Reply by AnzhiZhang** [reply]
支持增加后台按页面查看评论，请求开一个issue

**Reply by lizheming** [reply]
现在最新版前台已经支持管理评论了，可以考虑直接进前台页面管理

**Comment by YiHui-Liu** [comment]
![waline](https://user-images.githubusercontent.com/30190271/164514653-af74ff83-2ef0-4318-8068-e9d70c69d2eb.gif)

**Reply by YiHui-Liu** [reply]
环境是@waline/vercel:^1.13.0+MySQL8.0

**Reply by lizheming** [reply]
执行如下 SQL 修改下表结构吧，之前的表结构有点问题，会自动更新 `insertedAt` 字段

`alter table `wl_Comment` change `insertedAt` `insertedAt` timestamp NOT NULL default CURRENT_TIMESTAMP;`

**Reply by YiHui-Liu** [reply]
🆗修复了
